catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-right/navigation-right.hbs

```diff
@@ -11,17 +11,17 @@
  *
  **/
  --}}
-<button class="navigation-item item-help is-button" title="Shows helpful hints in the current context.">
-    <span class="fa fa-question"></span>
+<button class="navigation-item navigation-icon item-help is-button" title="Shows helpful hints in the current context.">
+    <span class="fa fa-question-circle"></span>
```

Comment:
Why the change to the circle?

Reply:
Just the way the requirement was passed down to me in the ticket. I believe it conforms to the way-forward UX mockups from @TeresaHudson and @michaelmenousek .